### PR TITLE
Changed format_version default value to 4

### PR DESCRIPTION
**Pull request by jiaqizho**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

- Changed `format_version` default value to 4.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@jiaqizho Could you please invite a technical reviewer? Thanks!

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -66,8 +66,9 @@ delta_index_cache_size = 0
     ## DTFile 储存文件格式
     ## * format_version = 1 老旧文件格式，已废弃
     ## * format_version = 2 v6.0.0 以前版本的默认文件格式
-    ## * format_version = 3 v6.0.0 及以后版本的默认文件格式，具有更完善的检验功能
-    # format_version = 3
+    ## * format_version = 3 v6.0.0 版本的默认文件格式，具有更完善的检验功能
+    ## * format_version = 4 v6.1.0 及以后版本的默认文件格式，优化了写放大问题和后台线程消耗更少。
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:70]:
```suggestion
    ## * format_version = 4 v6.1.0 及以后版本的默认文件格式，优化了写放大问题，同时减少了后台线程消耗
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

**Review by JaySon-Huang — APPROVED** [review]
LGTM

**Pull request merged**